### PR TITLE
feat: Remove LLVM submodule

### PR DESCRIPTION
**Pull request by jubianchi**

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -95,9 +72,13 @@ jobs:
     name: Release
 
     runs-on: ubuntu-latest
-    if: needs.setup.outputs.DOING_RELEASE == '1'
+    if: startsWith(github.ref, 'refs/tags/v')
```
Comment (Hywan) [inline .github/workflows/ci.yml:75]: :+1:

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -95,9 +72,13 @@ jobs:
...
 
-    needs: [setup, build]
+    needs: [build]
+
+    strategy:
+      matrix:
+        llvm_version: ['10.x', '11.x']
```
Comment (Hywan) [inline .github/workflows/ci.yml:81]: Can't we use YAML references here? (I know, yes…)

**Review by Hywan — APPROVED** [review]
Looks good to me!

File: .github/workflows/ci.yml (hunk 2 above)
Comment (jubianchi) [reply .github/workflows/ci.yml:81]: Github workflows do not support YAML anchors yet: https://github.com/jubianchi/llvm-custom-builds/actions/runs/381596227

**Pull request merged**